rust/hyperlane-core/src/traits/signing.rs

```diff
@@ -126,3 +137,52 @@ impl<T: Signable + Debug> Debug for SignedType<T> {
         )
     }
 }
+
+mod hashes {
```

Comment:
nit: could you leave a comment saying this was borrowed from https://github.com/hyperlane-xyz/ethers-rs/blob/hyperlane/ethers-core/src/utils/hash.rs so that we can get EIP-191 hashing without the `ethers` feature